src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -24,31 +24,55 @@ const PhotoDescription = (
   </div>);
 
 const MIN_SIZE = 350;
-const MIN_RATIO = 0.2;
```

Comment:
Since `viewMode={0}` does not constrain the movement or zooming of the canvas, the `minRatio` must be dynamic- it's used to maintain the zoom boundary. 

Both ratios were moved onto the state for consistency.